frontend/components/vault/WithdrawDaiFromVat.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>{{ currentStateAndTitle.title }}</template>
+        <TextBlock v-if="isExplanationsShown">
+            After liquidating a vault, the DAI-based liquidation incentive will end up in your VAT account. One more
+            transaction is required to move that DAI to your wallet.
+        </TextBlock>
+        <div class="my-4">
+            <WalletAuthorizationCheckPanel
+                :wallet-address="walletAddress"
+                :is-wallet-authorized="isWalletAuthorized"
+                :is-explanations-shown="isExplanationsShown"
+                :is-loading="isAuthorizing"
+                @authorizeWallet="$emit('authorizeWallet')"
+            />
+        </div>
+        <div class="flex justify-end mt-2 gap-5">
+            <BaseButton
+                :disabled="!isWalletAuthorized || !hasDaiToWithdraw || isWithdrawing"
+                @click="$emit('manageVat')"
+            >
+                Manage DAI in VAT
+            </BaseButton>
+            <BaseButton
+                type="primary"
+                :disabled="!isWalletAuthorized || !hasDaiToWithdraw"
+                :is-loading="isWithdrawing"
+                @click="$emit('withdrawAllDaiFromVat')"
+            >
+                <span v-if="hasDaiToWithdraw">
+                    Withdraw <FormatCurrency :value="daiVatBalance" currency="DAI" /> from VAT
+                </span>
+                <span v-else> Nothing to withdraw yet </span>
+            </BaseButton>
+        </div>
+    </BasePanel>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import BigNumber from 'bignumber.js';
+import { VaultTransactionState } from 'auctions-core/src/types';
+import BaseButton from '~/components/common/inputs/BaseButton.vue';
+import TextBlock from '~/components/common/other/TextBlock.vue';
+import FormatCurrency from '~/components/common/formatters/FormatCurrency.vue';
+import BasePanel from '~/components/common/other/BasePanel.vue';
+import WalletAuthorizationCheckPanel from '~/components/panels/WalletAuthorizationCheckPanel.vue';
+
+export default Vue.extend({
+    name: 'WithdrawDAIPanel',
+    components: {
+        BaseButton,
+        BasePanel,
+        TextBlock,
+        FormatCurrency,
+        WalletAuthorizationCheckPanel,
+    },
+    props: {
+        walletAddress: {
+            type: String,
+            default: undefined,
```

Comment:
These are all still set with a default value. My recommendation was to set these to required.

```suggestion
            required: true,
```